lib/shared/messages.js

```diff
@@ -52,6 +52,13 @@ module.exports = {
       'You are currently flashing a drive. Closing Etcher may leave',
       'your drive in an unusable state.\n\n',
       'Are you sure you want to close Etcher?'
+    ].join(' ')),
+
+    looksLikeWindowsImage: _.template([
+      'It looks like you are trying to write a Windows image.',
+      'Windows images require special treatment that is not currently built into Etcher in order to be made bootable.',
+      'It is recommended to use a specialised tool such as <a href="#" os-open-external="https://rufus.akeo.ie">Rufus</a>',
```

Comment:
I'll reply here to https://github.com/resin-io/etcher/pull/1043/files#r98021509
> OK, looks not, but I have an alternative :) Can we register a global even handler for anchors and intercept that using `OSOpenExternalService`? This could be a `.run()` block in `lib/gui/os/open-external/open-external.js`.

Yes, I thought of something on that lines also, but for some reason (probably because I wanted to see if it works 🙂) I decided to go for `os-open-external`. Now that I think about it again, I like your suggestion more because it separates presentation from behaviour. I'll give it a try.